src/reader/animation/animation.js

```diff
@@ -55,13 +55,6 @@ class Animation extends Component {
       audio.innerHTML =
         "<source src=\"" + file + ".ogg\" type=\"audio/ogg\"/>" +
         "<source src=\"" + file + ".m4a\" type=\"audio/mp4\"/>"
-      promises.push(new Promise(resolve => {
-        audio.oncanplaythrough = () => resolve(true)
-        audio.onerror = () => resolve(true)
-        let first = true
-        audio.firstElementChild.onerror = () => first ? (first = false) : resolve(true)
-        audio.lastElementChild.onerror = () => first ? (first = false) : resolve(true)
-      }))
```

Comment:
I had a feeling that was the case. However, the resource is failing to load, but it does not resolve the promise causing the element to just not load at all. Right now, it just doesn't work for iOS devices, and I think a usable work around could be to just not load this for iOS devices and check the user agent. TBH, when going through the game, it was barely noticeable that there was any differences because the assets don't even play until after a touch start event.

Reply:
01e26896a68a259b746adc461122321d1464e6dc